components/console-greet.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import InlineScript from './inline-script'
+
+export default ConsoleGreet
+
+function ConsoleGreet() {
+  return <InlineScript fn={greet} />
+}
+
+function greet() {
+  const CONTRIBUTING =
+    'https://github.com/paypal/glamorous/blob/master/CONTRIBUTING.md'
+  const USERS = 'https://github.com/paypal/glamorous/blob/master/other/USERS.md'
+
+  const GLAMOROUS =
+    ' ██████╗ ██╗      █████╗ ███╗   ███╗ ██████╗ ██████╗  ██████╗ ██╗   ██╗███████╗\n' +
+    '██╔════╝ ██║     ██╔══██╗████╗ ████║██╔═══██╗██╔══██╗██╔═══██╗██║   ██║██╔════╝\n' +
+    '██║  ███╗██║     ███████║██╔████╔██║██║   ██║██████╔╝██║   ██║██║   ██║███████╗\n' +
+    '██║   ██║██║     ██╔══██║██║╚██╔╝██║██║   ██║██╔══██╗██║   ██║██║   ██║╚════██║\n' +
+    '╚██████╔╝███████╗██║  ██║██║ ╚═╝ ██║╚██████╔╝██║  ██║╚██████╔╝╚██████╔╝███████║\n' +
+    ' ╚═════╝ ╚══════╝╚═╝  ╚═╝╚═╝     ╚═╝ ╚═════╝ ╚═╝  ╚═╝ ╚═════╝  ╚═════╝ ╚══════╝\n'
+
+  const gradientFont = `
+    font-size: 20px; font-family: "Helvetica",sans-serif;
+    background: -webkit-linear-gradient(top, #FFB6C1, #ED5C70);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+  `
+
+  /* eslint-disable */
+  console.log("%c💄 Welcome to 💄", "font-size: 24px; color: #444;")
+
+  console.log(`%c${GLAMOROUS}`, "color: #ED5C70;")
+
+  console.log("%c\n👉 We want you to contribute 👈", gradientFont)
+  console.log(`%c${CONTRIBUTING}`, "font-size: 14px")
+
+  console.log(
+    "%c\n😎 If you are using glamorous already please add it here 😎",
+    gradientFont,
```

Comment:
Because this code will actually not be transpiled (see [the implementation](https://github.com/kentcdodds/glamorous-website/blob/master/components/inline-script.js)) we need to make sure that the code will run in browsers we want to support (pretty much just the latest stuff). That means no fun stuff like comma-dangle on function parameter lists and calls (like this). Sad day.

I'm pretty sure that prettier is responsible for adding this. There are a few ways to avoid this. One of them is to assign things to variables so the function call lines are shorter.

On second thought... Maybe it does get transpiled 🤔 

If it does get transpiled, is there any reason you've got the `eslint-disable` lines?